packages/web/src/pages/upload-page/fields/DatePickerField.tsx

```diff
@@ -0,0 +1,48 @@
+import 'react-dates/initialize'
+import 'react-dates/lib/css/_datepicker.css'
+
+import { useState } from 'react'
+
+import cn from 'classnames'
+import { useField } from 'formik'
+import moment from 'moment'
+import { SingleDatePicker, isInclusivelyBeforeDay } from 'react-dates'
+
+import { ReactComponent as IconCalendar } from 'assets/img/iconCalendar.svg'
+
+import styles from './DatePickerField.module.css'
+
+type DatePickerFieldProps = {
+  name: string
+  label: string
+  style?: string
+}
+
+export const DatePickerField = (props: DatePickerFieldProps) => {
+  const { name, label, style } = props
+  const [field, , helpers] = useField(name)
```

Comment:
ah does it complain if you do the underscore thing? this hurts my eyes, but may be what we agreed to do